src/main/java/mate/academy/internetshop/controllers/CartProductRemoveController.java

```diff
@@ -0,0 +1,25 @@
+package mate.academy.internetshop.controllers;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.academy.internetshop.lib.Injector;
+import mate.academy.internetshop.service.ProductService;
+import mate.academy.internetshop.service.ShoppingCartService;
+
+public class CartProductRemoveController extends HttpServlet {
```

Comment:
```suggestion
public class RemoveProductFromCartController extends HttpServlet {
```

Reply:
I made this naming because it's faster for me to find everything that is related to cart etc, can I leave it or it's better to fully change it?

Reply:
Done